packages/commercetools/theme/components/Checkout/UserBillingAddresses.vue

```diff
@@ -77,11 +78,20 @@ export default {
 };
 </script>
 
-<style>
-  .billing__addresses {
-    display: grid;
-    grid-template-columns: 1fr 1fr;
-    margin-bottom: var(--spacer-xl);
+<style lang="scss">
```

Comment:
As you're already refactoring classess. Maybe you could modify name `billing-address-setAsDefault` to match BEM too? (there is need to change in both template and stylings)